### ISSUE TITLE
Vortex refuses to launch

### ISSUE THREAD

**Issue by Flood123789** [pikdum/steam-deck#3]
I been using your tool since I had gotten my deck a few weeks ago, when to refresh my game and re installed your tool from scratch along with it. Vortex refuses to launch after nuking all files from your app. Factory reset steamdeck and the issue persists. I noticed one possible issue here: 
![20230419_051824](https://user-images.githubusercontent.com/73972594/233036443-b6e83c3b-75ad-47ce-8127-45f2e0b444fb.jpg)
 I seem to have an access denied error that I'm not sure how to correct. Can I get your assistance?

**Comment by alechartung** [comment]
What release channel is your steam deck on?
That error shouldn't be the issue, but this might be related to https://github.com/pikdum/steam-deck/issues/2.

I probably need to do a fresh install on my system to be able to reproduce.

**Comment by pikdum** [comment]
What release channel is your steam deck on?
That error shouldn't be the issue, but this might be related to https://github.com/pikdum/steam-deck/issues/2.

I probably need to do a fresh install on my system to be able to reproduce.

**Comment by Flood123789** [comment]
i am currently on the stable channel

**Comment by pikdum** [comment]
@Flood123789 Should have a fix:

1. completely remove the broken Vortex:
```shell
rm -rf ~/stl/
rm -rf ~/.config/steamtinkerlaunch/
rm -rf ~/.local/share/applications/pikdum-vortex.desktop
```
2. run 'Update Pikdum's Tools' on desktop
3. run 'Install Vortex' on desktop

Issue was caused by a pre-release Vortex 1.8.0 version, which is crashing on startup here.
This quick fix just forces Vortex 1.7.8 to be installed instead.

I've opened https://github.com/sonic2kk/steamtinkerlaunch/issues/792 to track Vortex 1.8.0 support and/or an upstream fix.

Let me know if that fixes things. :)

**Comment by Flood123789** [comment]
That did the trick! Thank you for the help!

**Comment by pikdum** [comment]
No problem, thanks for reporting the issue.
I'll go ahead and close this issue then.

**Comment by aerojet029** [comment]
when closing Vortex, it autoupdates even with the setting checked not to update, forcing it to break itself. any way to force a newer version of proton or experimental?

**Comment by pikdum** [comment]
> any way to force a newer version of proton or experimental?

Yes, but couldn't get working versions of Proton GE to work at all on Steam Deck for whatever reason.

**Comment by aerojet029** [comment]
using Proton Expiremental through steam seems to have the vortex manager working fine, Been trying a ton of things to get it to work through launching from steam, but the "deployment" of mods is just completly borked. also I'm not sure how to create an "excecutable" to work when downloading from Fireforx

**Comment by pikdum** [comment]
Just shipped a fix that should have Vortex 1.8 working.
Probably need to re-install to be safe, though.
